services/galley/src/Galley/API/Teams/Features.hs

```diff
@@ -670,10 +675,9 @@ instance GetFeatureConfig db ValidateSAMLEmailsConfig where
 instance SetFeatureConfig db ValidateSAMLEmailsConfig where
   setConfigForTeam tid wsnl = persistAndPushEvent @db tid wsnl
 
-instance GetFeatureConfig db DigitalSignaturesConfig where
-  -- FUTUREWORK: we may also want to get a default from the server config file here, like for
-  -- sso, and team search visibility.
-  getConfigForServer = pure defFeatureStatus
+-- FUTUREWORK: we may also want to get a default from the server config file here, like for
+-- sso, and team search visibility.
```

Comment:
```suggestion
```

(it's true that that may be the case, but it seems arbitrary that we point this out, and not a million other similar things.)

Reply:
Yes, 100%. I just moved the comment, I didn't add this. However, removing this is fine.